var/spack/repos/builtin/packages/libpeas/package.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+import os
+
+
+class Libpeas(AutotoolsPackage):
+    """libpeas is a gobject-based plugins engine, and is targeted at
+    giving every application the chance to assume its own
+    extensibility."""
+
+    homepage = "http://developer.gnome.org/libpeas/stable"
+
+    version('1.22.0',
+            url='https://download.gnome.org/sources/libpeas/1.22/libpeas-1.22.0.tar.xz',
```

Comment:
Add this `url` as a package-level URL so that `spack versions` and `spack checksum` can find new versions. Then add a `url_for_version` below all the depends_on:
```python
def url_for_version(self, version):
    url = 'https://download.gnome.org/sources/libpeas/{0}/libpeas-{1}.tar.xz'
    return url.format(version.up_to(2), version)
```

Reply:
@adamjstewart Done.